### PR TITLE
[FE] feat: chatting, voice, note, drawboard의 isShared 수정 기능 추가.

### LINKED ISSUE #126: [FE] feat: chatting, voice, note, drawboard의 isShared 수정 기능 추가.
## 구현 기능
 chatting, voice, note, drawboard의 isShared 수정 기능 추가.
<br/>

## 작업 내용

<br/>

### PR DESCRIPTION
**Pull request by Y-minseong**
## 주요 변경사항
- chatting, voice, note, drawboard의 isShared 수정 기능 추가
## 리뷰어에게...
- Switch 위치가 맘에 들지 않다면 편하게 말씀해주세요 !
## 관련 이슈

- closes #126

### CODE REVIEW

**Comment by loopy-lim** [comment]
https://code-sync.net/s/255753ae-2ff8-4eaa-b55b-58b3bed4d3b4

File: src/components/Conversation/Save/Chatting.tsx [hunk 1]
```diff
@@ -39,8 +50,30 @@ const ConversationSaveChatViewer = ({
...
+              {voiceIsShared ? "공개" : "비공개"}
+            </span>
+            <div className="my-2">
+              <Switch
+                checked={voiceIsShared}
+                onCheckedChange={setVoiceIsShared}
+              />
+            </div>
+            <span className="text-2xl font-bold">Voice Text</span>
+          </div>
```
Comment (loopy-lim) [inline src/components/Conversation/Save/Chatting.tsx:76]: 하나로 합칠 수 있지 않을까요?

**Comment by loopy-lim** [comment]
https://code-sync.net/s/a6cc3db4-26af-4aed-beb2-1774ad98a063

**Comment by loopy-lim** [comment]
https://code-sync.net/s/858f1cde-1cd1-431d-93f0-fa15784fd571

File: src/global.d.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+interface Window {
+    SpeechRecognition: typeof SpeechRecognition;
+    webkitSpeechRecognition: typeof SpeechRecognition;
+  }
```
Comment (loopy-lim) [inline src/global.d.ts:4]: 이거 필요 없는거 아닌가요?

**Comment by loopy-lim** [comment]
https://code-sync.net/s/174bdd99-4de3-4edd-b78d-1aac78b1c29a

**Comment by loopy-lim** [comment]
https://code-sync.net/s/596a3b20-321d-400b-ac1c-497859d668ed

**Comment by loopy-lim** [comment]
https://code-sync.net/s/1edaea33-31d9-4015-a47d-a8119f09d90e

**Comment by HyoungUkJo** [comment]
https://code-sync.net/s/30e76317-643f-4bcc-8df2-19ee803bf13c

**Comment by HyoungUkJo** [comment]
https://code-sync.net/s/7db1fea4-bb63-42f2-ab62-685162d3cd3c

**Comment by HyoungUkJo** [comment]
https://code-sync.net/s/25d2bae0-9d31-480d-bc93-cf7c6cfb5fa8

**Pull request merged**